IdentityCore/src/util/ios/UIApplication+MSIDExtensions.m

```diff
@@ -36,7 +36,16 @@ + (UIViewController *)msidCurrentViewController:(UIViewController *)parentContro
     
     if ([MSIDAppExtensionUtil isExecutingInAppExtension]) return nil;
     
-    for (UIWindow *window in [MSIDAppExtensionUtil sharedApplication].windows)
+    NSArray<UIScene *> *scenes; NSArray<UIWindow *> *windows;
+    
+    scenes = [[[MSIDAppExtensionUtil sharedApplication] connectedScenes] allObjects];
+    
+    if (scenes && scenes.count != 0)
+    {
+        windows = [(UIWindowScene *)[scenes objectAtIndex:0] windows];
+    }
+    
+    for (UIWindow *window in windows)
```

Comment:
Seems we're using scenes and converting back to windows. Does this work on all platforms? Can we keep the old code and have an #if check for visionOS as this may result in different behavior.

Reply:
+1, seems this will impact core flows everywhere in addition to VisionOS as well

Reply:
You're right, this should definitely be only scoped for visionOS. Will update. For context, this why the change is needed at all:
`'windows' is deprecated: first deprecated in visionOS 1.0 - Use UIWindowScene.windows on a relevant window scene instead`